ratelimit/http.go

```diff
@@ -38,58 +38,54 @@ type middleware struct {
 
 // WithLocalRateLimiter returns a middleware that controls the amount of requests per tenant using an in-memory store.
 func WithLocalRateLimiter(configs ...Config) Middleware {
-	middlewares := make(map[string][]middleware)
```

Comment:
FYI these changes seem to break rate limiting, since it looks like only the last config will be applied. Maybe it would be worthwhile to tests for rate limiting as well.

Reply:
Ack. As discussed lets add more tests to cover rate limiting functionality, for now tests are passing.